app/src/main/java/com/example/android/sunshine/app/ForecastAdapter.java

```diff
@@ -75,17 +75,18 @@ public void bindView(View view, Context context, Cursor cursor) {
         ViewHolder viewHolder = (ViewHolder) view.getTag();
 
         int viewType = getItemViewType(cursor.getPosition());
+        int weatherId = cursor.getInt(ForecastFragment.COL_WEATHER_CONDITION_ID); // TODO bad practice!?!?!
```

Comment:
more explanatory comments will always help yourself when you come back to the code :-)